.resinci.yml

```diff
@@ -0,0 +1,41 @@
+---
+npm:
+  environment:
+    - name: CCACHE_TEMPDIR
+      value: /tmp/.ccache-temp
+    - name: CCACHE_COMPRESS
+      value: 1
+    - name: CC
+      value: clang
+    - name: CXX
+      value: clang++
+  platforms:
+    - name: linux
+      os: ubuntu
+      architecture: x86_64
+      node_versions:
+        - "10"
+        - "8"
+        - "6"
+    - name: linux
+      os: ubuntu
+      architecture: x86
+      node_versions:
+        - "8"
+        - "6"
```

Comment:
Why no Node v10 on x86? Just to reduce build count, or is there another reason?

Reply:
Node v10 doesn't support x86. https://github.com/nodejs/node/issues/20751